composer.json

```diff
@@ -20,7 +20,18 @@
 		"johnbillion/query-monitor": "^3.7.1",
 		"altis/dev-tools-command": "^0.4.0",
 		"wp-phpunit/wp-phpunit": "5.8.1",
-		"phpunit/phpunit": "^7.5.20"
+		"phpunit/phpunit": "^7.5.20",
+		"lucatume/wp-browser": "^3.0",
```

Comment:
We should use full Semver versions as much as possible, tracking only patch releases for a given package means less can change for a given major version of Altis. Not doing this in the past has caused some surprises that were tricky to debug. In general for 3rd party stuff I prefer the tilde constraint so `~3.0.13` in this case.

Same for the Codeception modules below.